pkg/scheduler/capabilities/volumebinding/binder.go

```diff
@@ -1010,12 +1013,6 @@ func (b *volumeBinder) revertAssumedPVCs(claims []*v1.PersistentVolumeClaim) {
 // hasEnoughCapacity checks whether the provisioner has enough capacity left for a new volume of the given size
 // that is available from the node.
 func (b *volumeBinder) hasEnoughCapacity(logger klog.Logger, provisioner string, claim *v1.PersistentVolumeClaim, storageClass *storagev1.StorageClass, node *v1.Node) (bool, error) {
-	// This is an optional feature. If disabled, we assume that
-	// there is enough storage.
-	if !b.capacityCheckEnabled {
```

Comment:
Was this code deleted by mistake?
**b.csiDriverLister** and **b.csiStorageCapacityLister** is null when **capacityCheckEnabled == false**

Reply:
yes, it's fixed now.